packages/webcert/src/store/certificate/certificateReducer.ts

```diff
@@ -85,6 +84,8 @@ const initialState: CertificateState = {
   routedFromDeletedCertificate: false,
   functionDisablers: [],
   clientValidationErrors: [],
+  createdCertificateId: '',
+  navigateToCertificate: false,
```

Comment:
This isn't used and can be removed, right?

Reply:
I remove navigateToCertificate.